src/app/components/Overview/components/Tables/Operators/Operators.tsx

```diff
@@ -43,73 +42,51 @@ const Operators = () => {
     SsvNetwork.getInstance().fetchOperators({ page: 1, validatorsCount: 'true' })
       .then((result: any) => {
         overviewStore.setTotalOperators(result.data.pagination.total);
+        setOperatorsExist(result.data.operators.length > 0);
```

Comment:
Better to use `pagination.total` as well

Reply:
done